kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF/mii-initial-d1-JRF-v1.yaml

```diff
@@ -115,6 +115,7 @@ spec:
                       values:
                         - $(CLUSTER_NAME)
                 topologyKey: "kubernetes.io/hostname"
+    # The number of managed servers to start for unlisted clusters
```

Comment:
This comment is incorrect. It should be 'The number of managed servers to start for this cluster'.